### ISSUE TITLE
Readme.md example code requires bitcore

### ISSUE THREAD

**Issue by seetd** [bitpay/bitcoind-rpc#30]
It looks like line `var bitcore = require('bitcore');` in the Readme.md example is unused. Should that be removed. I can put in the PR

**Comment by seetd** [comment]
Looking more into the code I guess it is required

**Comment by carnesen** [comment]
@seetd It seems to me that these days the "bitcore" package just exports the contents of the "bitcore-lib" package https://github.com/bitpay/bitcore/blob/95dc8b291089e2dec814d9a762df2ea341edef40/index.js So either way works fine. You're right though that "bitcore-lib" is probably "more right".

**Comment by seetd** [comment]
That's true. The reason I brought that back up was bitcore encompasses a lot more than bitcore-lib. So  I thought that making it use bitcore-lib will be more modular if you are not using bitcore. I would be glad to create a PR for that if no one disagrees with that.
